libs/utils/trace_analysis.py

```diff
@@ -970,138 +1030,140 @@ def getClusterFrequencyResidency(self, cluster):
             return None
 
         freq_dfr = self.trace.df('cpu_frequency')
```

Comment:
For consistency within this file, can we rename all the dataframe variables to be either just "df", when only one is in scope, or name_df when needed. For example this should be freq_df.